src/types/weatherData.ts

```diff
@@ -25,4 +24,5 @@ export type WeatherData = {
     winddir: number;
     weather_condition_id: object;
     weather_station_id: WeatherStation;
+    weather_condition: string;
```

Comment:
I think we should convert properties using camelCase - if you agree I'm happy to do it as part of my refactoring ticket.

Reply:
Hm, if we go down this road, you probably need to change a couple more things. The API response, maybe props here and there. Are you sure? I have no problem with it and I do like camelCase more. Up to you tbh :)

Reply:
Let me have a think about it, not because of the work required, but to check a few things so we can align with best practices. I'll mark this as resolved for now since it's not meant to block this PR, just wanted your opinion on that